packages/inscription-view/src/components/widgets/tag/Tags.test.tsx

```diff
@@ -9,16 +9,20 @@ describe('Tags', () => {
   } {
     let tags: string[] = [];
     userEvent.setup();
-    const view = render(<Tags tags={['test', 'bla']} onChange={newTags => (tags = newTags)} />, {
-      wrapperProps: {
-        meta: {
-          tags: ['demo', 'deprecated']
+    const view = render(
+      <Tags customValues={true} tags={['test', 'bla']} availableTags={['test', 'bla']} onChange={newTags => (tags = newTags)} />,
+      {
+        wrapperProps: {
+          meta: {
+            tags: ['demo', 'deprecated']
+          }
         }
```

Comment:
since you have moved the useMeta hook out of the Tag component, I think you don't have to add this meta information to the render function